### DISCUSSION TITLE
Chrome headless uses 90% cpu ussage. Is there any way to use Firefox?

### DISCUSSION THREAD

**Discussion by Dobbosynths** [kaliiiiiiiiii/Selenium-Driverless#65] [Q&A] [answered]

**Comment by ttlns** [comment, marked as answer]
Nope, actually this isn't easely possible. I might have a look if it actually uses to much cpu tho:)

**Reply by Dobbosynths** [reply]
ok thanks

**Comment by ttlns** [comment]
@Dobbosynths Can you provide:
1. What OS you run on
2. an example script you used to test
3. what specs (RAM, CPU) your device has?
4. What your device model is?

I only get maximum about 15% CPU-usage with:
- single-tab
- Moving the mouse constantly (automated)
- Windows 10 Laptop

**Reply by Dobbosynths** [reply]
I am on kali linux. I cant remeber the script i was on it was a while ago but i remember it went only on one website and i am on a asus vivobook
